tests/integration_tests/functional/test_cmd_line_parameters.py

```diff
@@ -24,8 +24,12 @@ def test_describe_snapshot_all_versions(bin_cloner_path):
     # Fetch all firecracker binaries.
     # For each binary create a snapshot and verify the data version
     # of the snapshot state file.
+
+    # TODO: FIXME: Once we upload 1.0.0 binaries in
+    # S3 that include io_uring, replace the
+    # max_version with get_version_from_toml()
     firecracker_artifacts = artifacts.firecrackers(
-        max_version=get_firecracker_version_from_toml())
+        max_version="0.25.0")
```

Comment:
This can create problems. We might forget to do this or not understand why the tests are failing before the release. Could we add a binary build from this branch instead of the current 1.0.0 binary ? Would the tests pass on the main branch ?

Reply:
Another folder would complicate things. It's true that uploading a binary build from this branch wouldn't be ideal either. But as long as it doesn't break the functionality from the main branch personally I would prefer the latter. Anyway I'm curious what others think too.

I will try to post a PR for adding snapshotting support as soon as possible. Anyway I ran some manual tests and I didn't encounter problems. In general it should work.

Apart from this the PR LGTM.

Reply:
ok I suggest we merge the PR as is and upload binaries and re-enable the tests after the snapshotting support is finalised, since we'll have to re-upload the artifacts anyway. sounds good?

Reply:
yes, sounds good to me.